### DISCUSSION TITLE
[MainMenu] Nested menu item has no hover effect. Is it a bug?

### DISCUSSION THREAD

**Discussion by not-SAINT** [epam/UUI#1478] [Q&A] [answered]
Hi here.
We found out that hover color in nested menu item is the same for normal stage after lib upgrade up to latest version.
background-color: var(--uui-main_menu-dropdown-bg) //  #1d1e26  - normal state
background-color: var(--uui-main_menu-hover) // #1d1e26  - hover state
![image](https://github.com/epam/UUI/assets/60464016/9ca68991-6679-4aed-8fa5-1a66ffd41bee)
Previous version (4.*.*) has values: 
background-color: var(--dropdown-body-background-color) // #1d1e26  - normal state
background-color: var(--dropdown-item-background-color) // #303240 - hover state 
![image](https://github.com/epam/UUI/assets/60464016/4b799ca6-5c84-45db-9a61-30aa81b033c0)

Same state [here](https://uui.epam.com/documents?id=mainMenu&mode=doc&skin=UUI4_promo&category=components).

Is this a bug? Or now hover color doesn't highlight menu items intentionally?

**Comment by AlekseyManetov** [comment, marked as answer]
Hi, yes, it's a bug. I've created the corresponding issue - https://github.com/epam/UUI/issues/1485. Thanks.
